packages/web/src/components/layouts/SearchResults/SearchResults.tsx

```diff
@@ -13,8 +13,32 @@ import CardList from '../../ui/CardList/CardList';
 import CardGroups from 'components/ui/CardGroups/CardGroups';
 import SidebarCategory from '../../ui/SidebarCategory/SidebarCategory';
 import { Button } from '../../ui';
+import LoadingBar, { LoadingBarRef } from 'react-top-loading-bar';
+import SidebarMeta from '../../ui/SidebarMeta/SidebarMeta';
+
+type Props = {
+  pageLoading?: boolean;
+};
+
+export default function SearchResults({ pageLoading = false }: Props) {
+  const [loading, setLoading] = useState<boolean>(pageLoading);
+  const loadingRef = useRef<LoadingBarRef>(null);
+
+  // FIXME: just for demo purposes to show how loading bar works
+  // Documentation: https://github.com/klendi/react-top-loading-bar
+  // Starts the loading indicator with a random starting value between 20-30 (or startingValue),
+  // then repetitively after an refreshRate (in milliseconds), increases it by a random value
+  // between 2-10. This continues until it reaches 90% of the indicator's width.
+  useEffect(() => {
+    loadingRef?.current?.continuousStart(10, 5000);
```

Comment:
nit: We'll likely want a redux middleware which controls this based on async thunks and a separate top-level loading bar component.

Reply:
Agree. LoadingBar is likely to reside in the top-level app component.

Reply:
I propose leaving it as is for demo purposes, before actual data could be used. Maybe we could replace this component with our own solution or perhaps use more redux-compatible library/package.